src/SonarQube.Client/SonarQubeService.cs

```diff
@@ -46,14 +40,15 @@ public class SonarQubeService : ISonarQubeService, IDisposable
         private readonly ISecondaryIssueHashUpdater secondaryIssueHashUpdater;
         private readonly ISSEStreamReaderFactory sseStreamReaderFactory;
 
-        private HttpClient httpClient;
+        private const string MinSqVersionSupportingBearer = "10.4";
```

Comment:
is it 10.4 or 10.5?

Reply:
10.4:
https://github.com/SonarSource/sonarlint-core/blob/07d22bc61ebe07f41ae42706d08d2b54a439c311/backend/server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/ServerVersionAndStatusChecker.java#L32